apps/element-storybook/src/migration.mdx

```diff
@@ -81,16 +81,18 @@ The migration theme is accessible by passing `theme='legacyBS'` to the `ThemePro
 #### Recommended Early Component Swaps
 
 - layout (`Box`, `Grid`, `Container`)
-- button?
+- button
```

Comment:
It seems safer to move this to the no drop-in section given the pattern changes. or at least until we have button migration docs that outline the pattern changes and how to reconcile them.

Reply:
good point, moved